pkg/controller/elasticsearch/driver/nodes.go

```diff
@@ -151,26 +155,26 @@ func (d *defaultDriver) reconcileNodeSpecs(
 		return results.WithError(err)
 	}
 	if requeue {
-		results.WithResult(defaultRequeue)
+		results.WithReconciliationState(defaultRequeue.WithReason("Not enough available masters to update Zen1 settings"))
 	}
 	// Remove the zen2 bootstrap annotation if bootstrap is over.
 	requeue, err = zen2.RemoveZen2BootstrapAnnotation(ctx, d.Client, d.ES, esClient)
 	if err != nil {
 		return results.WithError(err)
 	}
 	if requeue {
-		results.WithResult(defaultRequeue)
+		results.WithReconciliationState(defaultRequeue.WithReason("Zen2 bootstrap is not complete"))
```

Comment:
I think Zen2 is not an official term. Maybe we should avoid surfacing this to users. How about `Initial cluster bootstrap is not complete` or something like this.